### DISCUSSION TITLE
Issue with OpenAM upgrade from 13.5.2 to 14.6.4 - Entry cn=schema cannot be modified

### DISCUSSION THREAD

**Discussion by PJahnavi** [OpenIdentityPlatform/OpenAM#446] [Q&A]
Upon upgrade with OpenAM.war(14.6.4) on Tomcat 7.0.99, we are facing below issue.  Could anyone please help with this.

amUpgrade:11/29/2021 03:24:18:444 AM PST: Thread[http-bio-8080-exec-12,5,main]: TransactionId[935faa65-8514-4fda-b0f7-e8dddaf6da60-30]

ERROR: An error occurred while processing /WEB-INF/template/ldif/sfha/cts-add-multivalue.ldif

org.forgerock.opendj.ldap.ConstraintViolationException: No Such Attribute: Entry cn=schema cannot be modified because the attempt to update attribute objectclasses would have removed one or more values from the attribute that were not present: ( 1.3.6.1.4.1.36733.2.2.2.27 )

**Comment by vharseko** [comment]
please apply to OpenDJ schema file: https://github.com/OpenIdentityPlatform/OpenAM/blob/da05a2f102161c5cbbc091ad52f34a3c2cd6ee68/openam-server-only/src/main/webapp/WEB-INF/template/ldif/sfha/cts-add-schema.ldif

**Reply by PJahnavi** [reply]
@vharseko 
Could you please explain your suggestion to apply this ldif file. The ldif  which errored out is "cts-add-multivalue.ldif". Thanks in advance!

**Reply by PJahnavi** [reply]
@vharseko 

Upon commenting "delete" in /WEB-INF/template/ldif/sfha/cts-add-multivalue.ldif upgrade is moving past the above error, 

<img width="947" alt="image" src="https://user-images.githubusercontent.com/19340344/150362008-13b29e33-1438-41fc-86ee-9c813c3a1640.png">


but now it stuck at below task in amUpgrade log. 

amUpgrade:01/20/2022 06:38:57:023 AM PST: Thread[http-bio-8080-exec-9,5,main]: TransactionId[1c6602bf-5420-4dd1-84a0-520787a02ef3-52]
Found schema xml attribute changes for sunAMAuthAdaptiveService. Applying now.
